parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -181,6 +199,17 @@ def __init__(self, *,
         self.monitoring_queue = self._mp_manager.Queue()
         self.pending_task_queue = SpawnContext.Queue()
         self.pending_result_queue = SpawnContext.Queue()
+        self.task_scheduler: NoopScheduler
```

Comment:
this signature could look nicer if the naming didn't imply that a task_scheduler always did nothing -- eg a different base class?